sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -78,4 +79,16 @@ export default abstract class HttpBaseClient {
     this.handleRequestError(error);
     throw error;
   }
+
+  // TODO: ONCE ALL URL'S ARE UPDATED, THEN THIS
+  // FUNCTION CAN BE A PRIVATE FUNCTION AND CALLED
+  // BY getCallTyped OR PATCH OR POST FUNCTION
+  public addHook(url: string) {

```

Comment:
Nice job. 👍

But this hook is specific to AEST, so either we can change the method name like AESTHook or pass the hook string as argument.

Reply:
no, its not specific to `aest`, it can grow, when we implement all other modules, just add the case here

Reply:
maybe i will update the comment

Reply:
the method implementation had aest hardcoded, thats why I thought so and asked if it can be passed as an argument.

Reply:
@ann-aot, can we add the others ones in this to be already defined?
I would assume that would be below ones, right? @dheepak-aot, @guru-aot, @henryoforeh-aot 
- aest
- institutions
- students
- supporting-users